src/overlays/actors/ovl_Obj_Hana/z_obj_hana.c

```diff
@@ -27,11 +26,73 @@ const ActorInit Obj_Hana_InitVars = {
     (ActorFunc)ObjHana_Update,
     (ActorFunc)ObjHana_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Hana/ObjHana_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Hana/ObjHana_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 8, 10, 0, { 0, 0, 0 } },
+};
+
+static CollisionCheckInfoInit sColChkInfoInit = { 0, 0xC, 0x3C, 0xFF };
+
+typedef struct {
+    /* 0x00 */ Gfx* dList;
+    /* 0x04 */ f32 scale;
+    /* 0x08 */ f32 yOffset;
+    /* 0x0C */ s16 radius;
+    /* 0x0E */ s16 height;
+} HanaParams; // size 0x10
+
+HanaParams D_80B93AA4[] = { { 0x05000500, 0.01f, 0.0f, -1, 0 },
+                            { 0x0500A880, 0.1f, 58.0f, 10, 18 },
+                            { 0x0500B9D0, 0.4f, 0.0f, 12, 44 } };
```

Comment:
you can call the data symbol hanaParams or even flowerParams if you like. 
And a trick for the formatter, if you put a trailing comma on the last entry it will stop it from indenting all the way out like this.